### ISSUE TITLE
Trouble setting timeout global config

### ISSUE THREAD

**Issue by alexcroox** [se-panfilov/mini-toastr#97]
I'm using mini-toaster in one of my single file components in vue2.

I have the following:

```
<script>
    import miniToastr from 'mini-toastr'

    export default {

        mounted () {
            miniToastr.init({
                timeout: 10000,
            })

            miniToastr.setIcon('success', 'i', {'class': 'fa fa-check-circle-o'})

            miniToastr.success('message')
        },
```

It correctly shows a success message with a tick icon, however it disappears after 3 seconds, no matter the value I put for `timeout`

**Comment by se-panfilov** [comment]
@alexcroox I'll check&fix it soon

**Comment by se-panfilov** [comment]
@alexcroox Issue fixed at [v0.6.5](https://github.com/se-panfilov/mini-toastr/releases/tag/v0.6.5)

**Comment by alexcroox** [comment]
Thanks!
